module/kvmfr.c

```diff
@@ -41,6 +41,11 @@ DEFINE_IDR(kvmfr_idr);
 #define KVMFR_DEV_NAME    "kvmfr"
 #define KVMFR_MAX_DEVICES 10
 
+static int static_size_mb[KVMFR_MAX_DEVICES];
+static int static_count;
+module_param_array(static_size_mb, int, &static_count, 0000);
+MODULE_PARM_DESC(static_size_mb, "List of static devices to create in MiB (= 1048576 bytes)");
```

Comment:
Is `(= 1048576 bytes)` really required? `MiB` is pretty explicit.

Reply:
Removed.